### PR TITLE
Implement Table component and planning screen

### LINKED ISSUE #8: Unmock Table Component
As requested by @themaxhero on #7 I am creating this issue because the component is mocked, but we will still merge for the sake of routing.

### LINKED ISSUE #9: Unmock Planning Screen
As requested by @themaxhero on #7 I am creating this issue because the component is mocked, but we will still merge for the sake of routing.

### PR DESCRIPTION
**Pull request by Amadeusvh**

### CODE REVIEW

File: src/App.test.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
```
Comment (themaxhero) [inline src/App.test.tsx]: Please remove this scaffolding test.

File: src/components/Table.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                                <td>Cartão</td>
+                                <td>R$ 4000</td>
+                            </tr>
+                        </tbody>
+                    </Table>
+                </Col>
+            </Row>
+        </Container>
+    )
+}
```
Comment (themaxhero) [inline src/components/Table.tsx:73]: Please, Open an Issue about unmocking this component.

File: src/pages/Planning.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            </Row>
+            
+            <Row>
+                <Col>
+                    <Table />
+                </Col>
+            </Row>
+        </Container>
+    )
+}
```
Comment (themaxhero) [inline src/pages/Planning.tsx:51]: Please, Open an Issue about unmocking this component.

**Review by themaxhero — CHANGES_REQUESTED** [review]

File: src/components/Table.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
+import { Col, Container, Row, Table } from "react-bootstrap"
+import "./Table.css"
+
+type Props = {
```
Comment (jvrmalv) [inline src/components/Table.tsx:4]: Remember to type props

**Review by jvrmalv — APPROVED** [review]

**Review by themaxhero — APPROVED** [review]

**Review by Merzarus — APPROVED** [review]

**Pull request merged**